File: client_mobile_app/lib/models/Attachment.dart


class Attachment {
  String? id;
  String idReport;
  String nameFile;
  String path;
  String ext;
  Attachment({
    this.id,
    required this.idReport,
    required this.path,
    required this.nameFile,
    required this.ext,
  });
  Map<String, dynamic> toMap() => {
        "id": id,
        "idReport": idReport,
        "nameFile": nameFile,
        "path": path,
        "ext": ext,
      };
  factory Attachment.fromMap(Map<String, dynamic> map) {
    return Attachment(
        id: map["id"],
        idReport: map["idReport"],
        path: map["path"],
        nameFile: map["nameFile"],
        ext: map["ext"]);
  }
}

File: Server_ASP.NET/Data/db/Report/Attachment.cs
using System.ComponentModel.DataAnnotations;

namespace Security_Management_Server.db
{
    public class Attachments
    {
        [Key]
        [MaxLength(50)]
        public string Id { get; set; }
        [Required]
        [MaxLength(50)]
        public string id_Report { get; set; }
        public string nameFile { get; set; }
        public string ext { get; set; }
    }
}
